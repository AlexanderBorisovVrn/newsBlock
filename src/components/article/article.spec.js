import React, { Children } from 'react';
import Article from './article';
import { shallow } from 'enzyme';

const setUp = (props)=>shallow(<Article>{{}}</Article>)

describe('should render Article component',()=>{
  it('it should contain .article',()=>{
    const comp =setUp()
    expect(comp).toMatchSnapshot();
  })
})

