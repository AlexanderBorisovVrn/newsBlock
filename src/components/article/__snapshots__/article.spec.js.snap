// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Article component it should contain .article 1`] = `
<article
  className="article"
>
  <div
    className="wrap"
  >
    <div
      className="article__inner"
    >
      <h1
        className="article__title"
      />
      <p
        className="article__author"
      />
      <div
        className="article__img-wrap"
      >
        <img
          alt=""
          className="article__img"
          height="500"
          width="1200"
        />
      </div>
      <div
        className="article__txt"
      >
        <p />
      </div>
    </div>
  </div>
</article>
`;
